packages/utils/src/io.ts

```diff
@@ -39,19 +39,29 @@ export function readFileSync(path: string): string {
 /** If a file doesn't exist, warn and tell the step it should have been generated by. */
 export async function readFileAndWarn(generatedBy: string, filePath: string): Promise<object> {
   try {
-    return await readJson(filePath);
+    return await readJson(filePath, isObject);
   } catch (e) {
     console.error(`Run ${generatedBy} first!`);
     throw e;
   }
 }
 
-export function readJsonSync(path: string): object {
```

Comment:
Pro tip, `JSON.parse` does not always produce an `object` 🤦 

Quality of life enhancements throughout this section